client/logs/bdqmpvjoo3sunlgvpre0.log

```diff
@@ -0,0 +1,18 @@
+Applications
```

Comment:
I don't think these files supposed to be under version control. Missing a `.gitignore` directive?